test/config/config-defaults.yaml

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-defaults
```

Comment:
🤔 What do we need this for?

Reply:
https://github.com/knative/net-istio/pull/4/files#diff-58de452513b7e8d8d3cfea23eb4ae6a8R34

For defaulting.  I am wondering this is also where we need config-autoscale